### ISSUE TITLE
The bundle targets unknown languages: [cb].

### ISSUE THREAD

**Issue by andrespaez90** [bitrise-steplib/steps-google-play-deploy#61]
I get this error message uploading a new bundle, how I can solve that?

**Comment by Roland-Bak** [comment]
Hi there @andrespaez90 👋

Thanks for reaching out!

We might need a bit more information surrounding this error message, could you please send us the log file or the URL of the related build and also enable Support Access on the Settings tab of the app ( https://devcenter.bitrise.io/troubleshooting/enabling-bitrise-support-user/ ) so that we can take a more in-depth look? 🙂

**Comment by nilscox** [comment]
Hello!

Is there any update on this?

I followed the second method provided in [this SO thread](https://stackoverflow.com/questions/57875775/invalid-splitapkbundle-the-bundle-target-unknown-language-gr), by updating my build-extras.gradle this way:

```
android {
    ...
    bundle {
        density {
            enableSplit true
        }
        abi {
            enableSplit true
        }
        language {
            enableSplit false
        }
    }
}
```

This seems to fix the issue, but I must admit I don't fully understand what it means.

**Comment by andrespaez90** [comment]
Hi!, 

I've already solved, I defined "resConfigs" on the gradle, apparently, there is a library that has a res folder with that suffix.

thanks

**Comment by anka-213** [comment]
This suddenly happened to me as well. I tried to make a new build of the exact same version of the code and the second build failed with this error `Failed to upload app bundle, error: googleapi: Error 403: The bundle targets unrecognized languages: fb., apkNotificationMessageKeyBundleTargetsInvalidLanguage
`. The only thing I had changed was adding a secret with an API key for the Google Places API.
